lightning/src/ln/channelmanager.rs

```diff
@@ -10452,8 +10452,13 @@ where
 			.map(|(node_id, _)| *node_id)
 			.collect::<Vec<_>>();
 
+		let receive_tlvs = message::ReceiveTlvs {
+			context: Some(context),
+			custom_data: None,
```

Comment:
We'll want to let users set this, too, right? Probably best to do in a follow-up after #3412 is landed, but we will eventually need ways for them to add it to offer/refund message paths and reply paths (e.g., when paying for an offer). Let's update the PR description to mention what is done in this PR and what will be done in a follow-up.

Reply:
Makes sense!
I’ve added a "TODO" comment in the code in [pr2830.11](https://github.com/shaavan/rust-lightning/commits/pr2830.11) and updated the PR description accordingly.

Thanks so much for pointer!